changelogs/fragments/4457-support-datadog-monitors-type event-v2.yaml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
```

Comment:
Ah, one thing: this is more a new feature than a bugfix, so it should be `minor_changes`.
```suggestion
minor_changes:
```

Reply:
updated